public/app/features/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplatesTable/AlertRuleTemplatesTable.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useState } from 'react';
+import { useSortBy, useTable } from 'react-table';
+import { Spinner, useStyles } from '@grafana/ui';
+import { getStyles } from './AlertRuleTemplatesTable.styles';
+import { css } from 'emotion';
+import { logger } from '@percona/platform-core';
+import { AlertRuleTemplateService } from '../AlertRuleTemplate.service';
+import { formatTemplates } from './AlertRuleTemplatesTable.utils';
+import { FormattedTemplate } from './AlertRuleTemplatesTable.types';
+import { Messages } from './AlertRuleTemplatesTable.messages';
+
+export const AlertRuleTemplatesTable = () => {
+  const style = useStyles(getStyles);
+  const [pendingRequest, setPendingRequest] = useState(false);
+  const [data, setData] = useState<FormattedTemplate[]>([]);
+
+  const getAlertRuleTemplates = async () => {
+    setPendingRequest(true);
+    try {
+      const { templates } = await AlertRuleTemplateService.list();
+      setData(formatTemplates(templates));
+    } catch (e) {
+      logger.error(e);
+    } finally {
+      setPendingRequest(false);
+    }
+  };
+
+  const columns = React.useMemo(
+    () => [
+      {
+        Header: 'Name',
+        accessor: 'summary',
+        width: '70%',
+      },
+      {
+        Header: 'Source',
+        accessor: 'source',
+        width: '20%',
+      },
+      {
+        Header: 'Created',
+        accessor: 'created_at',
+        width: '10%',
+      },
+    ],
+    []
+  );
+
+  useEffect(() => {
+    getAlertRuleTemplates();
+  }, []);
+
+  const tableInstance = useTable({ columns, data }, useSortBy);
```

Comment:
I guess the sorting is out of scope at this stage. Could be useful though, be we haven't decided which columns to sort and which not. Also, design-wise we need a standard approach, i.e. the user should be able to see which column(s) is(are) sorted.

Reply:
you're right, removing it

Reply:
done